src/app/containers/Blocks/index.jsx

```diff
@@ -8,21 +11,34 @@ const BlockString = props => {
   return <p>{stringProps}</p>;
 };
 
+const UnrecognisedContent = styled.div`
+  font-family: ${FF_NEWS_SANS_REG};
+  background: ${C_LUNAR};
+  padding-top: 1rem;
+  padding-left: 0.5rem;
+`;
+
 const Blocks = ({ blocks, componentsToRender }) =>
-  blocks.map((block, index) => {
+  blocks.map(block => {
     const { type, model } = block;
+    const subblocks = block.model.blocks;
+
+    if (Object.keys(componentsToRender).includes(type)) {
+      const Block = componentsToRender[type] || BlockString;
+
+      if (type === 'caption') {
```

Comment:
I see why you want to keep our block passing consistent! Totally want to avoid this conditional 👍

Reply:
Haha yep. It's a spike 🤫😉